src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
```

Comment:
Be more specific with variable names, something like `inversedRobot` would be a perfect fit here